samples/matter/common/src/bridge/ble_connectivity_manager.cpp

```diff
@@ -212,6 +215,17 @@ void BLEConnectivityManager::DisconnectionHandler(bt_conn *conn, uint8_t reason)
 	}
 }
 
+#ifdef CONFIG_BRIDGE_SET_BT_CONNECTION_PARAMS
+/* this callback accept or decline conn params requested by peripheral. */
+bool BLEConnectivityManager::ParamChangeRequestHandler(struct bt_conn *conn, struct bt_le_conn_param *param)
+{
+	LOG_ERR("requested param %d", param->interval_max);
+	LOG_ERR("requested param %d", param->interval_min);
+	return (param->interval_max >= CONFIG_BRIDGE_BT_CONNECTION_INTERVAL_MAX &&
```

Comment:
Looks neat! Could you add comment explaining that we prefer to use bigger intervals to reduce radio time usage by BLE?